test/e2e/lib/gutenberg/blocks/layout-grid-block-component.js

```diff
@@ -7,11 +7,94 @@ import { By } from 'selenium-webdriver';
  * Internal dependencies
  */
 import GutenbergBlockComponent from './gutenberg-block-component';
+import * as driverHelper from '../../driver-helper';
 
 class LayoutGridBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Layout Grid';
 	static blockName = 'jetpack/layout-grid';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-jetpack-layout-grid' );
+
+	/**
+	 * Setups the number of columns for the layout grid block.
+	 *
+	 * This needs to be called before you call @see {@link insertBlock}.
+	 *
+	 * @param {number} no number of columns, 1 to 4 (based on the default buttons in the block).
+	 */
+	async setupColumns( no ) {
+		const columnButtonSelector = By.css( `${ this.blockID } button[aria-label="${ no } columns"]` );
+		await driverHelper.clickWhenClickable( this.driver, columnButtonSelector );
+
+		// Updates the blockId, since the block is replaced by another one upon the selection of the columns.
+		this.blockID = await this.driver
+			.findElement( By.css( 'div.block-editor-block-list__block.is-selected' ) )
+			.getAttribute( 'id' );
+	}
+
+	/**
+	 * Inserts a block in the last available grid for the layout grid.
+	 *
+	 * You must call @see {@link setupColumns} before inserting an inner block using this method.
+	 *
+	 * This will add to each column in sequence, from left to right.
+	 *
+	 * @param { typeof GutenbergBlockComponent } blockClass A block class that responds to blockTitle and blockName
+	 * @returns { GutenbergBlockComponent } instance of the added block
+	 **/
+	async insertBlock( blockClass ) {
+		const addColumnButtonsSelector = By.css(
+			`div[id="${ this.blockID }"] div.wp-block-jetpack-layout-grid button[aria-label="Add block"]`
+		);
+		const buttons = await this.driver.findElements( addColumnButtonsSelector );
+
+		await buttons[ 0 ].click();
+
+		const inserterSearchInputSelector = By.css( 'input.block-editor-inserter__search-input' );
```

Comment:
Looks like we can use the [`openBlockInserterAndSearch`](https://github.com/Automattic/wp-calypso/blob/712abcc43c2f719557db3f8cab380cb1d83d6443/test/e2e/lib/gutenberg/gutenberg-editor-component.js#L228) helper here?